luigi/task.py

```diff
@@ -289,6 +289,9 @@ def __init__(self, *args, **kwargs):
         self.task_id = task_id_str(self.task_family, self.to_str_params(only_significant=True))
         self.__hash = hash(self.task_id)
 
+        self.status_message = None
+        self._status_message_callback = None
```

Comment:
As the scheduler receives a request each time the status is changed/set, I think the ``status_message`` cache is required to prevent user errors, i.e., the possibility that the scheduler is flooded with too many requests.